tests/platform_tests/test_application.py

```diff
@@ -29,7 +29,7 @@ def setUp(self):
         
         # setup shot
         seq = {"type":"Sequence", "code": "seq_name", "id":3 }
-        seq_path = os.path.join(self.project_root, "sequences/Seq/seq_name")
+        seq_path = os.path.join(self.project_root, "sequences/seq_name")
```

Comment:
is this ok on windows (the slash?) or should it be `os.path.join(self.project_root, "sequences", "seq_name")